x-pack/metricbeat/module/prometheus/_meta/fields.yml

```diff
@@ -0,0 +1,31 @@
+- key: prometheus
+  title: "Prometheus"
+  description: >
+    Stats scraped from a Prometheus endpoint.
+  release: ga
```

Comment:
Will this be `ga`?

Reply:
The module is GA (it was already), this new feature will be beta/experimental. fields.yml is about the module, where docs & cfgwarn calls should warn about the beta status